src/html/header-basic.html

```diff
@@ -1,6 +1,40 @@
 
   <a class="usa-skipnav" href="#main-content">Skip to main content</a>
   <header class="usa-header" role="banner">
+    <!-- Gov banner BEGIN -->
```

Comment:
This gets added to our code snippets. I worry that it might be possibly be confusing to users when looking at the snipper. They might wonder if the skipnav above this line is also part of our example code.

Also, are we doing this anywhere else in our other code snippets?